apis/storage/v1alpha2/types.go

```diff
@@ -80,8 +81,8 @@ type AccountList struct {
 // An AccountClassSpecTemplate is a template for the spec of a dynamically
 // provisioned Account.
 type AccountClassSpecTemplate struct {
-	runtimev1alpha1.NonPortableClassSpecTemplate `json:",inline"`
-	AccountParameters                            `json:",inline"`
+	runtimev1alpha1.ClassSpecTemplate `json:",inline"`
+	AccountParameters                 `json:",inline"`
```

Comment:
Assuming `forProvider` will be a follow-up PR?

Reply:
Yeah, don't want to make that change in this already huge PR.